src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/SchemaCompareRequest.cs

```diff
@@ -70,6 +71,11 @@ public class SchemaCompareEndpointInfo
         /// Connection details
         /// </summary>
         public ConnectionDetails ConnectionDetails { get; set; }
+
+        /// <summary>
+        /// Extract target of the project used when extracting a database to file system or updating the project from database
+        /// </summary>
+        public DacExtractTarget ExtractTarget { get; set; }
```

Comment:
Let's get in the habit of being explicit about the nullability of properties :)  This can be `DacExtractTarget? ExtractTarget` since it may be null if it's a source or the target isn't a project, right?